app/controllers/catalog_controller.rb

```diff
@@ -345,24 +342,26 @@ def load_bulk_jobs(druid)
 
   # Determines whether or not the current user has permissions to view the current DOR object.
   def valid_user?(dor_object)
-    begin
-      @apo = dor_object.admin_policy_object
-    rescue
-      return false
-    end
-
+    # First check the workgroups defined for admin, manager or viewer; see
+    # User::ADMIN_GROUPS, User::MANAGER_GROUPS and User::VIEWER_GROUPS
+    return true if @user.is_admin || @user.is_manager || @user.is_viewer
+    # If that fails, check an APO for permitted roles of the current_user.
+    valid = false
+    @apo = dor_object.admin_policy_object rescue nil
     if @apo
-      unless @user.is_admin || @user.is_viewer || dor_object.can_view_metadata?(@user.roles(@apo.pid))
-        render :status => :forbidden, :text => 'forbidden'
-        return false
-      end
+      valid = dor_object.can_view_metadata?(@user.roles(@apo.pid))
+      msg = 'Governing APO forbids access' unless valid
+    elsif dor_object.is_a? Dor::AdminPolicyObject
+      # In the absence of a governing APO, allow an APO to authorize itself.
+      valid = dor_object.can_view_metadata?(@user.roles(dor_object.pid))
+      msg = 'Item is an APO that forbids access' unless valid
     else
-      unless @user.is_admin || @user.is_viewer
-        render :status => :forbidden, :text => 'No APO, no access'
-        return false
-      end
+      msg = 'Item has no APO that allows access'
+    end
+    unless valid
+      render :status => :forbidden, :text => msg
     end
-    true
+    valid
   end
 
   def get_leafdir(directory)
```

Comment:
Google Scanned books lack an APO since they pre-dated the concept.  That means in principle only sdr-* roles can find/see them since no APO filtering is applied to their views, and it's a condition we would like to see in Argo in order to fix it there.  However, new Argo still has a problem trying to display the item page for these (true, despite all the "let's not implode if a druid can't be de-referenced" work?) so we plan to add in the APO references for those before release.